twap/as/verify.go

```diff
@@ -22,6 +22,14 @@ type HostVerifyAttestationOutput struct {
 	RawClaims []byte `json:"raw_claims"`
 }
 
+const (
+	ATTEST_FN_CALL_HASH_OF_CODE_IDX    = 0
```

Comment:
In go, it is uncommon to use snake case for constants, just use cammel case with a capital if it is public or a lower case if it is private as you would do with any other constant.

For example: https://go.dev/doc/effective_go#parallel

Reply:
ok